src/sasjs-context/index.js

```diff
@@ -16,27 +16,74 @@ export async function processContext(command) {
   }
 
   if (!commands.hasOwnProperty(command)) {
-    console.log(chalk.redBright('Not supported context command.'))
+    console.log(
+      chalk.redBright(
+        `Not supported context command. Supported commands are:\n${Object.keys(
+          commands
+        ).join('\n')}`
+      )
+    )
 
     return
   }
 
-  const targetName = await getAndValidateTargetName()
+  const commandExample =
+    'sasjs context <command> -source ../contextConfig.json -target targetName'
```

Comment:
When passing parameters that are more then one letter, should it be prepended with 2 dashes? `--`

Reply:
@allanbowe should I change it to use `--`?

Reply:
fixed